### PR TITLE
mongo-optimization task Slava.telyatko

### PR DESCRIPTION
**Pull request by SlavaTel**
https://travis-ci.org/github/SlavaTel/db-assignments

### CODE REVIEW

File: task/mongo-optimization-task.js [hunk 1]
```diff
@@ -15,7 +15,14 @@ const ObjectId = require('mongodb').ObjectID;
...
  * */
 async function before(db) {
-    await db.collection('opportunities').createIndex({'initiativeId': 1});
+    await db.collection('opportunities').createIndex({
+        'initiativeId': 1,
+        "contacts.questions.category_id": 1, 
+        "contacts.datePublished": 1
+    });
+    await db.collection('clientCriteria').createIndex({
+        'value': 1
```
Comment (IlyaMokin) [inline task/mongo-optimization-task.js:24]: Compound index can work better here, but you need to re-work $lookup section to make it possible

Comment (SlavaTel) [reply task/mongo-optimization-task.js:24]: I upgraded lookup section and now time is about 3000 ms

**Comment by SlavaTel** [comment]
https://travis-ci.org/github/SlavaTel/db-assignments

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**